src/editor/codeeditor.h

```diff
@@ -77,15 +115,24 @@ public slots:
     QTimer *mTimer;
     LexerCPP mLexer;
     AddCommentButton *mAddCommentButton;
+    AddCommentTextEdit *mAddCommentTextEdit;
```

Comment:
Think, it`s neccessary to create AddCommentTextEdit here. You create it on CommentWidget (commentwidget.h, line 64)

Reply:
Just forgot to delete it, because in previous version I didn't have commentWidget and mAddCommentTextEdit was separate widget. After that I create mAddCommentTextEditas as a tab of commentWidget . Removed it